tools/ci/array-api-skips.txt

```diff
@@ -14,6 +14,6 @@ array_api_tests/test_signatures.py::test_func_signature[reshape]
 array_api_tests/test_signatures.py::test_func_signature[argsort]
 array_api_tests/test_signatures.py::test_func_signature[sort]
 
-# TODO: check why in CI `inspect.signature(np.vecdot)` returns (*arg, **kwarg)
-#       instead of raising ValueError. mtsokol: couldn't reproduce locally
+# ufuncs signature on linux is always <Signature (*args, **kwargs)>
+# np.vecdot is the only ufunc with a keyword argument which causes a failure
```

Comment:
Do the array API tests also check that vecdot's signature is correct using duck typing? Probably worth doing both - check the signature and validate that the signature is correct!

Reply:
There is `test_signatures.py` in array-api-tests suite for testing signatures. Separately, functions are tested by calling them with args+kwargs, like `test_vecdot` in `test_linalg.py`.